src/cli/commands/install.js

```diff
@@ -329,6 +329,12 @@ export class Install {
       return true;
     }
 
+    const {artifacts} = match;
```

Comment:
Is it the right place?
bailout() function is overridden in add.js.

In the next couple of weeks I'd like to do integrity check earlier in install phase.